tests/test.ts

```diff
@@ -367,20 +406,20 @@ test('is.class', t => {
 	];
 
 	for (const x of classDeclarations) {
-		t.true(m.class(x));
+		t.true(m.class_(x));
 	}
 });
 
 test('is.typedArray', t => {
 	const typedArrays = [
-		new Int8Array(),
-		new Uint8Array(),
-		new Uint8ClampedArray(),
-		new Uint16Array(),
-		new Int32Array(),
-		new Uint32Array(),
-		new Float32Array(),
-		new Float64Array()
+		new Int8Array(0),
```

Comment:
These can be created without a value, but I assume TS does not agree with that? If so, could you open a TS issue an link to it here?

See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Int8Array

Reply:
(added)]https://github.com/Microsoft/TypeScript/issues/19680], will add a comment in the code as well